packages/simcore-sdk/src/simcore_sdk/node_ports_v2/port.py

```diff
@@ -25,6 +25,7 @@ class Port(ServiceProperty):
     key: str = Field(..., regex=PROPERTY_KEY_RE)
     widget: Optional[Dict[str, Any]] = None
 
+    default_value: Optional[DataItemValue] = None
```

Comment:
one question here, are you not missing the alias defaultValue here?

Reply:
indeed ... tests made it crystal clear :-D